doc/source/whatsnew/v0.20.0.txt

```diff
@@ -41,6 +41,9 @@ Other enhancements
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
 
+- New ``UnsortedIndexError`` (subclass of ``KeyError``) thrown when indexing into an
+ unsorted index (:issue:'11897`)
```

Comment:
specificaly on MultiIndex? you are closing 2 issues, is there a second note (or you can list 2 for this one if applicable)